app/controllers/spree/taxons_controller_decorator.rb

```diff
@@ -9,24 +10,22 @@ def show
       end
 
       @taxon = Spree::Taxon.friendly.find(params[:id])
```

Comment:
We dnt need this line of code. As load_taxon function is already doing the same thing. Please justify this mistake.